components/rating.css

```diff
@@ -0,0 +1,43 @@
+:root{
+    --star-color: #ff9800;
+}
+
+.rating-card{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 1rem 2rem;
+    margin: 1rem auto;
+    box-sizing: border-box;
```

Comment:
its better to give *{box-sizing: border box } unless you are using content-box anywhere else in your library